docs/appendices/environment_vars.rst

```diff
@@ -217,6 +217,17 @@ There are several environment variables that affect the way Toil runs.
 |                                  | to S3 (``True`` by default).                       |
 |                                  | Example: ``TOIL_S3_USE_SSL=False``                 |
 +----------------------------------+----------------------------------------------------+
+| TOIL_FTP_USER                    | The FTP username to override all FTP logins with   |
+|                                  | Example: ``TOIL_FTP_USER=ftp_user``                |
++----------------------------------+----------------------------------------------------+
+| TOIL_FTP_PASSWORD                | The FTP password to override all FTP logins with   |
+|                                  | Example: ``TOIL_FTP_PASSWORD=ftp_password``        |
++----------------------------------+----------------------------------------------------+
+| TOIL_FTP_INSECURE                | Enable or disable connecting to all FTP            |
+|                                  | connections in insecure mode                       |
+|                                  | (``False`` by default).                            |
```

Comment:
It might be better to leave this as insecure by default instead because that was the default behavior before

Reply:
I don't know what "insecure mode" means.

Is the user going to need to set this if their FTP server doesn't support SSL? Or to bypass bad SSL certificates while non-SSL FTP servers still work fine?

Reply:
insecure in this case means FTP without SSL. Though I'm not sure if bypassing bad SSL certificates is supported